test/FrequencyMapAgentTest.cpp

```diff
@@ -332,12 +339,6 @@ TEST_F(FrequencyMapAgentTest, split_policy)
 #endif
 }
 
-TEST_F(FrequencyMapAgentTest, name)
-{
-    EXPECT_EQ("frequency_map", m_agent->plugin_name());
-    EXPECT_NE("bad_string", m_agent->plugin_name());
-}
-
```

Comment:
Why did this move up?  Please put it back.  It is not related to the other changes you are making.